aries_cloudagent/revocation/routes.py

```diff
@@ -1570,6 +1600,10 @@ async def register(app: web.Application):
                 get_active_rev_reg,
                 allow_head=False,
             ),
+            web.post(
+                "/revocation/active-registry/{cred_def_id}/decommission",
```

Comment:
I am wondering if the name of the endpoint should be changed to something like `rotate` rather than decommision, since this is the actual outcome of this action. It would make the intent clearer in my opinion.

Reply:
I like that. I think `decommissioned` is a good state name, but I like `rotate` for the API call.

Reply:
I agree — the state is right, but I like `rotate` for the API call.